File: Store/Store.Web/Modules/Store/WaresMovement/StoreAverageForm.cs

namespace Store.Store.Forms
{
    using Serenity.ComponentModel;
    using Serenity.Web;
    using System;

    [FormScript("Store.StoreAverage")]
    public class StoreAverageForm
    {
        [FileUploadEditor, Required]
        public String FileName { get; set; }
    }
}

File: Ser2023/Company/Company.Web/Modules/ServerTypes/Store/StoreAverageForm.ts
import { ImageUploadEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface StoreAverageForm {
    FileName: ImageUploadEditor;
}

export class StoreAverageForm extends PrefixedContext {
    static formKey = 'Store.StoreAverage';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!StoreAverageForm.init)  {
            StoreAverageForm.init = true;

            var w0 = ImageUploadEditor;

            initFormType(StoreAverageForm, [
                'FileName', w0
            ]);
        }
    }
}